samples/subsys/usb/uac2_explicit_feedback/src/main.c

```diff
@@ -36,7 +36,7 @@ LOG_MODULE_REGISTER(uac2_sample, LOG_LEVEL_INF);
  * when USB host decides to perform rapid terminal enable/disable cycles.
  */
 #define I2S_BUFFERS_COUNT   7
-K_MEM_SLAB_DEFINE_STATIC(i2s_tx_slab, MAX_BLOCK_SIZE, I2S_BUFFERS_COUNT, 4);
+K_MEM_SLAB_DEFINE_STATIC(i2s_tx_slab, MAX_BLOCK_SIZE, I2S_BUFFERS_COUNT, UDC_BUF_ALIGN);
```

Comment:
`MAX_BLOCK_SIZE` here has to be replaced with `ROUND_UP(MAX_BLOCK_SIZE, UDC_BUF_GRANULARITY)`.

Reply:
Yes, thank you. Maybe we should look closer at [Shared Multi Heap](https://docs.zephyrproject.org/latest/kernel/memory_management/shared_multi_heap.html) and replace slabs and static buffers with shared_multi_heap_alloc().

Reply:
Maybe, but here the memory slab is imposed by I2S API so the change cannot happen as freely as in the experimental USB stack.